tests/Dockerfile

```
@@ -1,9 +1,9 @@
 # syntax=docker/dockerfile:1.16
 # this is here so we can grab the latest version of kind and have dependabot keep it up to date
-FROM kindest/node:v1.33.4@sha256:25a6018e48dfcaee478f4a59af81157a437f15e6e140bf103f85a2e7cd0cbbf2
+FROM kindest/node:v1.34.0@sha256:7416a61b42b1662ca6ca89f02028ac133a309a2a30ba309614e8ec94d976dc5a
 
 # this is here so we can grab the latest version of skopeo and have dependabot keep it up to date
-FROM quay.io/skopeo/stable:v1.19.0@sha256:77e6c8901f6976bdfa87b8b14c40b96de754794c492d8ed295f77fba1454a4ab
+FROM quay.io/skopeo/stable:v1.20.0@sha256:85000771191808da4e142a61b66f7deb19fdca334b243341e2adc529b3087e8b
```

this is already out of date, I would use the v1.20.0 immutable tag listed here: https://quay.io/repository/skopeo/stable?tab=tags&tag=latest


```suggestion
FROM quay.io/skopeo/stable:v1.20.0@sha256:519b95573a7490eb4740c8180d3dc33d86417222956e655719e74d3509586a33
```

See https://quay.io/repository/skopeo/stable/manifest/sha256:519b95573a7490eb4740c8180d3dc33d86417222956e655719e74d3509586a33